assets/sass/dark-mode.css

```diff
@@ -152,6 +152,10 @@
             #l10n-versions-dropdown footer a {
                 color: #6969dd;
             }
+
+            svg.graphviz {
+                filter: invert(60%);
```

Comment:
I noticed this is different from the existing dark mode filter on the about/distributed page, which is `brightness(.9)contrast(.6)invert(1)`. I think the existing dark mode looks better.
<img width="542" height="208" alt="image" src="https://github.com/user-attachments/assets/222e0413-c3d8-49c2-95d3-3b5d89939122" />
<img width="380" height="170" alt="image" src="https://github.com/user-attachments/assets/2d112051-9e5f-4d3f-980c-bc9c8e3e1ac0" />